### ISSUE TITLE
Scam signposting

### ISSUE THREAD

**Issue by HazeyOneKenobi** [anon-real/ErgoAuctionHouse#50]
Add a user-removable banner to the main page detailing that scam NFT's are present on the marketplace, and how to avoid by checking mint address.

Alternatively add a mention of scams and how to avoid in FAQ. 

As it stands, without speaking to the community it is hard to find info on this anywhere on-site. It can't be expected that newcomers can avoid auctionhouse scams without any info present for them to do so.

**Comment by ergonauts1** [comment]
I agree that this is a low effort solution to bring attention to the fact that some eNFTS are not originals, and what steps can be taken to avoid purchasing them. I would also considering adding the duplicate checking tool: https://ergolui.com/nft-copy-check/
